Task/implement event messages

Jamie-Unosquare opened the pull request:
Event Messages Added
Simple functionality for adding and getting messages for General, Approval and Reject

nmcmahon-unosquare commented on v2/AdminCore.Common/Interfaces/IDashboardService.cs:
```diff
@@ -2,6 +2,7 @@
 using AdminCore.DTOs.Event;
 using System;
 using System.Collections.Generic;
+using AdminCore.DTOs.EventMessage;
```
Delete this

Jamie-Unosquare replied:
EventMessage has been moved out of Dashboard directory so it needs to be imported

nmcmahon-unosquare commented on v2/AdminCore.Services/EventMessageService.cs:
```diff
@@ -0,0 +1,67 @@
+    }
+
+    public void CreateGeneralEventMessage(int eventId, string message)
+    {
```
this method is a chonker, refactor into 3 methods: buildEventMessage, addEventMessageToEvent, InsertEventMessageIntoDb (or something like this)

nmcmahon-unosquare commented on v2/AdminCore.WebApi.Tests/Controllers/DashboardControllerTest.cs:
```diff
@@ -9,6 +9,8 @@
 using System.Collections.Generic;
 using System.Linq;
+using AdminCore.DTOs.EventMessage;
+using AdminCore.WebApi.Models.EventMessage;
```
revert file

Jamie-Unosquare replied:
Same as above, these imports are needed as EventMessage has been moved

Jamie-Unosquare committed:
Implement EventMessage Service
Simple Create and Get Event Messages
Checks if events already have event messages, if not it adds the property to prevent nulls,
Events show latest message

Jamie-Unosquare committed:
Merge branch 'develop' into Task/Implement-Event-Messages
